client/src/components/ui/Navbar.tsx

```
@@ -151,15 +143,33 @@
+          {/* Dropdown menu for the explore pages */}
+          <Menu as="div" className="relative">
+            <MenuButton className="inline-flex items-center px-3 py-2 text-sm font-medium text-gray-300 hover:bg-gray-700 hover:text-white rounded-md">
+              Explore
+              <svg className="ml-1 h-4 w-4" viewBox="0 0 20 20" fill="currentColor">
+                <path d="M5.23 7.21a.75.75 0 011.06.02L10 11.293l3.71-4.06a.75.75 0 111.08 1.04l-4.25 4.66a.75.75 0 01-1.08 0l-4.25-4.66a.75.75 0 01.02-1.06z" />
+              </svg>
+            </MenuButton>
+            <MenuItems className="absolute right-0 mt-2 w-48 origin-top-right rounded-md bg-white py-1 shadow-lg ring-1 ring-black/5 focus:outline-none z-10">
+              {subMenu.map((item) => {
+                return (
+                  <MenuItem>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)